rocAL_pybind/rocal_pybind.cpp

```diff
@@ -112,6 +145,13 @@ std::unordered_map<int, std::string> rocalToPybindOutputDtype = {
 
 PYBIND11_MODULE(rocal_pybind, m) {
     m.doc() = "Python bindings for the C++ portions of ROCAL";
+    // Bind the C++ structure
+    py::class_<ROIxywh>(m, "ROIxywh")
+        .def(py::init<>())
+        .def_readwrite("x", &ROIxywh::x)
+        .def_readwrite("y", &ROIxywh::y)
+        .def_readwrite("w", &ROIxywh::w)
+        .def_readwrite("h", &ROIxywh::h);
     // rocal_api.h
```

Comment:
please remove commented line

Reply:
The comments are added in the pybind to mention the APIs that are present in each rocal_api headers. The comments are made for all rocal_api headers